BrowserKit/Sources/MenuKit/HeaderBanner.swift

```diff
@@ -76,7 +76,8 @@ public final class HeaderBanner: UIView, ThemeApplicable {
         fatalError("init(coder:) has not been implemented")
     }
 
-    private func setupViews() {
+    private func setupViews(mainMenuHelper: MainMenuHelper = MainMenuHelper()) {
```

Comment:
The `MainMenuHelper` can never be overwritten for tests because both the `var` and this method are private. I think it would be better to make the `var` public and give it a default value. Additionally lets code against the interface `MainMenuInterface` instead of the implementation of the interface - this way we can swap out the implementation if needed in tests.